src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Cohort/ConfirmDelete.cshtml

```diff
@@ -0,0 +1,78 @@
+﻿@using SFA.DAS.EmployerUrlHelper.Mvc
+@using SFA.DAS.CommitmentsV2.Shared.Extensions
+@model SFA.DAS.EmployerCommitmentsV2.Web.Models.Cohort.ConfirmDeleteViewModel
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <partial name="_ValidationSummary" />
+        <h1 class="govuk-heading-xl">Confirm deletion</h1>
+        <form method="post">
+            <dl class="govuk-summary-list govuk-!-margin-bottom-7">
+                <div class="govuk-summary-list__row">
+                    <dt class="govuk-summary-list__key">
+                        Training provider
+                    </dt>
+                    <dd class="govuk-summary-list__value">
+                        @Model.ProviderName
+                    </dd>
+                </div>
+                <div class="govuk-summary-list__row">
+                    <dt class="govuk-summary-list__key">
+                        Reference
+                    </dt>
+                    <dd class="govuk-summary-list__value">
+                        @Model.CohortReference
+                    </dd>
+                </div>
+                <div class="govuk-summary-list__row">
+                    <dt class="govuk-summary-list__key">
+                        Number of apprentices
+                    </dt>
+                    <dd class="govuk-summary-list__value">
+                        @Model.DraftApprenticeshipsCount
+                    </dd>
+                </div>
+                <div class="govuk-summary-list__row">
+                    <dt class="govuk-summary-list__key">
+                        Apprentices
```

Comment:
According to the screen designs, this should say `Apprenticeships`

Reply:
good spot